Commandant/Switch.swift

```diff
@@ -59,6 +59,6 @@ public func <| <ClientError> (mode: CommandMode, option: Switch) -> Result<Bool,
 		return .success(enabled)
 
 	case .Usage:
-		return .failure(informativeUsageError(option.description, option.usage))
```

Comment:
Maybe we should remove the external label on the second parameter instead. I don’t have a strong opinion other than to note that `usage` is perhaps not a terribly helpful external label.